src/extension/background-script/index.ts

```diff
@@ -154,13 +161,23 @@ async function init() {
 }
 
 console.info("Welcome to Alby");
-init().then(() => {
-  if (isFirstInstalled && !state.getState().getAccount()) {
-    utils.openUrl("welcome.html");
-  }
-  if (isRecentlyUpdated) {
-    migrate();
-  }
-});
+init()
+  .then(() => {
+    if (resolveInit) {
```

Comment:
shouldn't resolveInit always exist here? can we throw an exception instead if it undefined?

Reply:
I think it's just for TS because `resolveInit?`, can't be changed to `resolveInit` because it's only set at runtime I guess.